components/teleport/Teleport.vue

```diff
@@ -168,19 +180,34 @@ const fetchBasiliskBalance = async () => {
     }
   )
 }
+
+const fetchKusamaBalance = async () => {
+  const api = await getKusamaApi()
+  subscribeBalance(api, getAddressByChain(Chain.KUSAMA), (...data) => {
```

Comment:
Could you pls tell me how to unsubscribe this rpc listener? I seem could not find the unsubscribe method in our code base.  @vikiival 

Not sure if it would help: https://github.com/polkadot-js/api/blob/master/packages/api-derive/src/balances/all.ts

Reply:
> Could you pls tell me how to unsubscribe this rpc listener? I seem could not find the unsubscribe method in our code base

It was automatically done by `utils/mixins/subscribeMixin.ts`

Reply:
Thanks! I have added it.